civis/ml/_model.py

```diff
@@ -1265,6 +1273,12 @@ def predict(self, df=None, csv_path=None,
             predict_args['LIMITSQL'] = sql_limit
         if n_jobs:
             predict_args['N_JOBS'] = n_jobs
+        if dvs_to_predict:
+            if isinstance(dvs_to_predict, six.string_types):
+                dvs_to_predict = [dvs_to_predict]
+            if self.predict_template_id > 10600:
```

Comment:
This is really nitpicky, but it would be clearer if this were 10583 instead of 10600, to match other parts of the code that reference the specific template id that is the cutoff for that argument.

Reply:
Good point. Changed.